backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -30,35 +32,40 @@
     private OrganizationRepository _repo;
     private FacilityRepository _facilityRepo;
     private ProviderRepository _providerRepo;
-    private ApiUserService _apiUserService;
     private AuthorizationService _authService;
     private OktaService _oktaService;
 
     public OrganizationService(OrganizationRepository repo,
             FacilityRepository facilityRepo,
             AuthorizationService authService,
             ProviderRepository providerRepo,
-            ApiUserService apiUserService,
             OktaService oktaService) {
         _repo = repo;
         _facilityRepo = facilityRepo;
         _authService = authService;
         _providerRepo = providerRepo;
-        _apiUserService = apiUserService;
         _oktaService = oktaService;
     }
 
-    public Organization getCurrentOrganization() {
+    public Optional<CurrentOrganizationRoles> getCurrentOrganizationRoles() {
```

Comment:
why is the CurrentOrganizationRoles class necessary? Does OrganizationRoles not suffice? worried that some dev could easily use the wrong one and confuse things.

Reply:
I don't love the naming, and would be happy to adjust either of them, but the difference is that `OrganizationRoles` has an external ID string that _might_ correspond to a known and non-deleted organization (and the user's roles), whereas `CurrentOrganizationRoles` has an actual Organization entity (and the user's roles). Deleting an org is mostly a hypothetical case now, but having it be supported in one branch of the code but not another seemed very bad.